projects/github-actions/repo-gardening/src/utils/parse-content/find-priority.js

```diff
@@ -2,37 +2,82 @@ const debug = require( '../debug' );
 
 /**
  * Figure out the priority of the issue, based off issue contents.
- * Logic follows this priority matrix: pciE2j-oG-p2
+ * Logic follows this priority matrix: pfVjQF-su-p2
  *
  * @param {string} body - The issue content.
  * @return {string} Priority of issue.
  */
 function findPriority( body ) {
+	let priority = 'TBD';
+
+	debug( `find-priority: Looking for priority indicators in issue body: ${ body }` );
+
 	// Look for priority indicators in body.
 	const priorityRegex =
-		/###\sImpact\n\n(?<impact>.*)\n\n###\sAvailable\sworkarounds\?\n\n(?<blocking>.*)\n/gm;
+		/###\sSite\sowner\simpact\n\n(?<impact>.*)\n\n###\sSeverity\n\n(?<severity>.*)\n\n###\sWhat\sother\simpact\(s\)\sdoes\sthis\sissue\shave\?\n\n(?<extra>.*)\n/gm;
 	let match;
 	while ( ( match = priorityRegex.exec( body ) ) ) {
-		const [ , impact = '', blocking = '' ] = match;
+		const { impact = '', extra = '' } = match.groups || {};
+		let { severity = '' } = match.groups || {};
+		const extras = extra.split( ', ' );
 
 		debug(
-			`find-priority: Reported priority indicators for issue: "${ impact }" / "${ blocking }"`
+			`find-priority: Reported priority indicators for issue: "${ impact }" / "${ severity }" / "${ extra }"`
 		);
 
-		if ( blocking === 'No and the platform is unusable' ) {
-			return impact === 'One' ? 'High' : 'BLOCKER';
-		} else if ( blocking === 'No but the platform is still usable' ) {
-			return 'High';
-		} else if ( blocking === 'Yes, difficult to implement' ) {
-			return impact === 'All' ? 'High' : 'Normal';
-		} else if ( blocking !== '' && blocking !== '_No response_' ) {
-			return impact === 'All' || impact === 'Most (> 50%)' ? 'Normal' : 'Low';
+		// Folks can provide additional information that can bump severity.
+		// We also do not want that extra information to downgrade the severity.
+		if ( extra !== '' && extra !== '_No response_' && ! extras.includes( 'No revenue impact' ) ) {
```

Comment:
If someone selects both "No revenue impact" and "Platform revenue" for some reason, I note it'll ignore the latter in favor of the former. Is that intended?

Reply:
That's an interesting use case. I don't expect anyone to pick those 2 options together, but given that "No revenue impact" is the last option in the field, I feel comfortable saying that it should top the other options you picked before that.